src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticListenerExtensions.cs

```diff
@@ -28,7 +28,7 @@ public static IDisposable SubscribeWithAdapter(this DiagnosticListener diagnosti
             Func<string, object, object, bool> isEnabled)
         {
             var adapter = new DiagnosticSourceAdapter(target, isEnabled);
-            return diagnostic.Subscribe(adapter, adapter.IsEnabled);
+            return diagnostic.Subscribe(adapter, (Predicate<string>)adapter.IsEnabled);
```

Comment:
.Subscribe has overloads in netcoreapp2.0/net461 that were not in ns1.3. The call was ambiguous.